### PR TITLE
Added settings CONSOLE_LOG and MINIFY_CONSOLE_LOG allowing log genera…

### LINKED ISSUE #19: log only to console log
Hi! 
I can make a pull request to create a version with a setting to disable the file logging (to only log to console)? It is ok?

### PR DESCRIPTION
**Pull request by limberger**
closes #19 
This Pull requests is to create to new settings: 
FILE_LOG (Default to True - When False don't  generate the file logger)
and
MINIFY_CONSOLE_LOG (Default to False - When True generate de console log in Json minified).
With defaults the behavior remains the same. 
With this changes i can use the django-logger in containers my infrastructure reads the log in  stderr. 
If you agree, I think it can be useful for more people.
Very Thank You!

### CODE REVIEW

**Comment by cipriantarta** [comment]
I moved away from Python and Django for a few years now, but I'm happy to merge this if you've thoroughly tested it and think it's working OK.

File: django_logging/logger.py [hunk 1]
```diff
@@ -7,20 +7,23 @@
 from . import settings
 
 LOG_LEVEL = settings.LOG_LEVEL.upper()
-LOG_HANDLERS = ['default']
+LOG_HANDLERS = []
```
Comment (cipriantarta) [inline django_logging/logger.py:10]: If the settings.*_LOG are all set to `false`, it means that nothing will get logged anymore, not sure if this is desirable?

Comment (limberger) [reply django_logging/logger.py:10]: Yes, but all settings must be explicitly set to false. 
If you prefer, it is possible to raise an exception in this case!

**Comment by limberger** [comment]
> I moved away from Python and Django for a few years now, but I'm happy to merge this if you've thoroughly tested it and think it's working OK.

I teste and it is working!

File: django_logging/logger.py (hunk 1 above)
Comment (limberger) [reply django_logging/logger.py:10]: Couldn't the option to mute the log be interesting?

Comment (cipriantarta) [reply django_logging/logger.py:10]: I guess the side effect can actually be considered a feature :)

**Comment by limberger** [comment]
Thanks!

**Pull request merged**